attackers/client.py

```diff
@@ -0,0 +1,50 @@
+import socket
+import response_handlers
+from create_update_request import update_request
+from common.const import *
+from common import protocol
+
+
+def connect_to_server(address: ADDR) -> socket.socket:
+    my_socket = socket.socket()
+    my_socket.connect(address)
+    return my_socket
+
+
+def create_request() -> FULL_REQUEST:
+    cmd = input(CMD_INPUT).lower()
+    if cmd == "update":
+        params = update_request()
+    else:
+        params = [input(PARAMS_INPUT).lower()]
+    return cmd, params
+
+
+def handle_response(response_cmd: str, response_params: str):
+    handler_function = getattr(response_handlers, response_cmd)
+    handler_function(response_params)
+    print("-" * 146)
+
+
+def is_client_disconnect(cmd: str) -> bool:

```

Comment:
A function that asks if a client disconected IMO should get a client and verify stuff.  
This function not only dosent get the client as a parameter it is not answering the questions.  
  
Currently this question that this function is answering is 'Is the command disconnects the client'.  
either change the name or change the implemnation.